yamale/tests/test_functional.py

```diff
@@ -210,6 +210,13 @@ def test_bad_schema():
     assert 'fixtures/bad_schema.yaml' in str(excinfo.value)
 
 
+def test_empty_schema():
```

Comment:
You should leave the previous test you had which tested the command line flow. Never get rid of valid tests. :)